### PR TITLE
testing/shadowsocks-libev: upgrade to 3.2.5

### PR DESCRIPTION
**Pull request by maxice8**

### CODE REVIEW

**Comment by maxice8** [comment]
@ncopa

File: testing/shadowsocks-libev/APKBUILD [hunk 1]
```diff
@@ -1,38 +1,41 @@
...
+license="GPL-3.0-or-later"
+makedepends="asciidoc autoconf automake libtool linux-headers libcorkipset-dev
+	mbedtls-dev pcre-dev xmlto libev-dev c-ares-dev libsodium-dev zlib-dev libbloom-dev"
+subpackages="$pkgname-dev $pkgname-doc lib$pkgname:libs"
 source="shadowsocks-libev-$pkgver.tar.gz::https://github.com/shadowsocks/shadowsocks-libev/archive/v$pkgver.tar.gz"
-builddir="$srcdir/shadowsocks-libev-$pkgver"
+
+prepare() {
+	default_prepare
+	sed -i 's|AC_CONFIG_FILES(\[libbloom/Makefile libcork/Makefile libipset/Makefile\])||' configure.ac
```
Comment (Ikke) [inline testing/shadowsocks-libev/APKBUILD]: I believe we prefer patches rather than inline sed statements for these kinds of things.

**Comment by maxice8** [comment]
On Sat, Jun 08, 2019 at 08:00:26AM -0700, Kevin Daudt wrote:
> Ikke commented on this pull request.
> 
> 
> 
> >  source="shadowsocks-libev-$pkgver.tar.gz::https://github.com/shadowsocks/shadowsocks-libev/archive/v$pkgver.tar.gz"
> -builddir="$srcdir/shadowsocks-libev-$pkgver"
> +
> +prepare() {
> +	default_prepare
> +	sed -i 's|AC_CONFIG_FILES(\[libbloom/Makefile libcork/Makefile libipset/Makefile\])||' configure.ac
> 
> I believe we prefer patches rather than inline sed statements for these kinds of things.
> 
> -- 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/alpinelinux/aports/pull/8572#pullrequestreview-247370807

If you insist, i copied from Arch Linux

**Comment by maxice8** [comment]
updated

**Pull request closed without merging**